mmpose/models/losses/mse_loss.py

```diff
@@ -42,6 +42,53 @@ def forward(self, output, target, target_weight):
         return loss / num_joints
 
 
+@LOSSES.register_module()
+class JointsMSELoss_Combined(nn.Module):
```

Comment:
name candidate: `CombinedJointsMSELoss`

Reply:
JointsMSELoss_CombinedTarget?

Reply:
Try again with

* avoid `_`
* end with `Loss`

Reply:
CombinedTargetJointsMSELoss? CombinedJoints is kind of strange, as 'Combined' is use to describe the targets instead of the joints

Reply:
`CombinedTargetMSELoss`?

Reply:
'CombinedTargetMSELoss' is better, 'Joints' in 'CombinedTargetJointsMSELoss' is quite unnecessary.

Reply:
Great